packages/react/emotion-utils/src/utils/position.ts

```diff
@@ -87,7 +99,8 @@ export function position(
 }
 
 function isPositionValue(value: unknown): value is Property.Position {
-  return [`static`, `relative`, `absolute`, `fixed`, `sticky`, `-webkit-sticky`].includes(value as string);
+  const positions: Property.Position[] = ['static', 'relative', 'absolute', 'fixed', 'sticky', '-webkit-sticky'];
+  return positions.includes(value as Property.Position);
```

Comment:
`value` is NOT `Property.Position`.

Reply:
That's right. I don't want to make a mistake this time, so I read code again.

I don't know why position function use `CSSProperties['position']`. Is there any other reason?
ex) next code is type safe.
```position(undefined, 1, 1, 1, 1)```

How about this code? 

```
type PositionOrTop = Property.Position | CSSPixelValue;
export function position(
  position: Property.Position,
  top: CSSPixelValue,
  right: CSSPixelValue,
  bottom: CSSPixelValue,
  left: CSSPixelValue,
): SerializedStyles;
export function position(
  top: CSSPixelValue,
  right: CSSPixelValue,
  bottom: CSSPixelValue,
  left: CSSPixelValue,
): SerializedStyles;
export function position(position: Property.Position, coordinates?: Coordinates): SerializedStyles;

export function position(
  positionOrTop: PositionOrTop,
  topOrRightOrCoordinates: CSSPixelValue | Coordinates = {},
  ...values: CSSPixelValue[]
) {
 // ...
}

function isPositionValue(value: PositionOrTop): value is Property.Position {
  return typeof value === 'string';
}
```

Reply:
That makes sense. It's a bug that the first parameter of `position` can be `undefined`.
So please fix the type of it to `Property.Position`.

But please keep the implementation of `isPositionValue` because `CSSPixelValue` also can be string.

Reply:
I miss again...

This code is not working in typescript.

```
function isPositionValue(value: PositionOrTop): value is Property.Position {
  const positions: Property.Position[] = ['static', 'relative', 'absolute', 'fixed', 'sticky', '-webkit-sticky'];
  return positions.includes(value);
}
```
![스크린샷 2022-10-25 오후 11 57 04](https://user-images.githubusercontent.com/57904979/197808689-e9159090-87c8-40c1-b6a2-136dff2dae72.png)

![스크린샷 2022-10-25 오후 11 35 42](https://user-images.githubusercontent.com/57904979/197803140-0d485920-18e5-460e-a9b6-3b1c171fa240.png)

So I wrote the code like this. I think it doesn't matter because value has type. But I don't know if you agree. What do you think about it?

Other ways
- use ts-ignore
- `positions.some((position) => position === value);`

```
  const positions: Property.Position[] = ['static', 'relative', 'absolute', 'fixed', 'sticky', '-webkit-sticky'];
  return positions.includes(value as Property.Position);
```

Reply:
?? I have already told you `value` is NOT `Property.Position`. Just keep the previous implementation of `isPositoinValue`.

It's the role of `isPositionValue` to determine that `value` is `Property.Position`.

```typescript
function isPositionValue(value: unknown): value is Property.Position {
  return ['static', 'relative', 'absolute', 'fixed', 'sticky', '-webkit-sticky'].includes(value as any);
}
```

Reply:
okey.. i understand

Reply:
Thanks for commenting. It was time to learn about typescript and toss library.